test/fixtures/rules/css-modules/undefined.js

```diff
@@ -0,0 +1,15 @@
+// undefined.js
+import React from 'react';
+
+import styles from './test.css';
+
+// `css-modules/no-undef-class` - Alerts whenever a CSS class is not being used
```

Comment:
I think the phrase is:  Alerts when using a CSS class that is not defined

Reply:
accidental copy paste   changed